docs/content/tips-and-tricks.md

```diff
@@ -34,11 +34,11 @@ There are a number of option flags that can be added to the `pk;system list` com
 ### Sorting options
 |Flag|Aliases|Description|
 |---|---|---|
-|-by-name|-bn|Sort by member name (default)|
+|-by-name|-bn|Sort by member/group name (default)|
```

Comment:
maybe a good idea to just remove member instead of adding group, I think "sort by name" is clear enough